### ISSUE TITLE
cdn.sockjs.org is gone

### ISSUE THREAD

**Issue by IlyaSemenov** [aio-libs/sockjs#152]
The [hardcoded](https://github.com/aio-libs/sockjs/blob/d7c7e3315c47143ed173e72ce57e693856f6e09d/sockjs/route.py#L31) `sockjs_cdn` URL http://cdn.sockjs.org/sockjs-0.3.3.min.js is no more valid:

```console
$ curl http://cdn.sockjs.org/sockjs-0.3.3.min.js
<?xml version="1.0" encoding="UTF-8"?>
<Error><Code>AccessDenied</Code><Message>Access Denied</Message><RequestId>3BB0BB487751DE22</RequestId><HostId>j8A9HZa9CXG2MqNWOX4ROjCuaCaDyrakWCQIYavU4o4lm6JXjpB0j7Epj81bkisFUu2lrki+dxo=</HostId></Error>
```

According to sockjs.org, the new CDN URL should be https://cdn.jsdelivr.net/sockjs/0.3/sockjs.min.js (or https://cdn.jsdelivr.net/sockjs/1.1/sockjs.min.js if 0.3 is not specifically required.)

**Comment by MartinKolarik** [comment]
The correct URL for newer versions (and works for the older ones too) is actually  https://cdn.jsdelivr.net/npm/sockjs-client@1.1/dist/sockjs.min.js

It's mentioned here: https://github.com/sockjs/sockjs-client#deployment. The link in "Getting started" section is outdated (I'll send a PR to fix that).

**Comment by asvetlov** [comment]
Please do

**Comment by asvetlov** [comment]
Fixed by 3f309f3
